<div class="step-text">
<p></p><p>There is a function <code class="java">suspiciousFunction</code>, and for some reason, you suspect that an exception might occur in it.</p><p>Process the call of <code class="java">suspiciousFunction</code> with the help of the <code class="java">try-catch-finally</code> statement. If an exception occurs in the function, print the exception message. </p><p>In the case of <code class="java">IOException</code>, print the following message instead: <code class="java">The IOException occurred</code>.</p><p>In the final block, output <code class="java">Handling completed successfully!</code>.</p><p><b>Tip:</b>  Use <code class="java">message</code> to get information about the exception. </p><br><br><font color="gray">Memory limit: 256 MB</font><br><font color="gray">Time limit: 8 seconds</font><br><br>
</div>